src/components/Grid/Grid.vue

```diff
@@ -144,10 +164,14 @@ export default {
 
     currentMode() { this.drawPolygons(); },
     polygons() { this.drawPolygons(); },
-
+    windowDefs: {
+      handler() { console.log('windowDefs was updated'); this.drawPolygons(); },
```

Comment:
did this actually need to be a deep watcher, or was the problem just caused by the ThermalZone typo?